plugwise/__init__.py

```diff
@@ -862,6 +866,30 @@ async def set_switch_state(
 
         await self._request(uri, method="put", data=data)
 
+    async def set_gateway_mode(self, mode: str) -> None:
+        """Set the gateway mode."""
+        if mode not in self._gw_allowed_modes:
+            raise PlugwiseError("Plugwise: invalid gateway mode.")
+
+        time_1 = dt.datetime.now(dt.UTC)
+        away_time = time_1.isoformat(timespec="milliseconds") + "Z"
+        time_2 = str(dt.date.today() - dt.timedelta(1))
+        vacation_time = time_2 + "T23:00:00.000Z"
+        end_time = "2037-04-21T08:00:53.000Z"
+        if mode == "away":
```

Comment:
This looks a bit weird, should we set `valid == ''` by default and then one if/else construction

Reply:
Implemented :)